style.css

```diff
@@ -0,0 +1,64 @@
+@media (min-width: 375px), (min-width: 768px) {
+  .Intro {
+    background-image: url("header-shapes\ mobile@2x.svg");
+  }
+
```

Comment:
- [ ] Great effort working on your mobile first portfolio project, but currently, I notice both the images and the background image in your project is not working and they are broken, I would kindly suggest you check the way you have named the images. 
My humber advice is to name your image without given space between the name

Kindly check the naming type
- Vector1.svg
- Vector2.svg
- Vector3.svg
- LinkedIn-icon.svg
- etc

I believe this way of naming the images will make your image appears in other people's machine

Your Design :point_down: 

![Screenshot_20230601_213928](https://github.com/Omwomo/Microverse-web-portfolio-project/assets/78634841/12b28b7e-2a32-4614-9522-fd73c3cf31b6)


- [ ] Also the color of the text, kindly be sure it matches the one in the Figma original design